### PR TITLE
LRAC-9035, LRAC-9036 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LRAC-9035
https://issues.liferay.com/browse/LRAC-9036

This is only for master.

@alexcandidos

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @Tim-Cao , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase [hunk 1]
```diff
@@ -27,9 +35,33 @@ definition {
...
+		}
+
+		task ("Assign the new site to existing property") {
+			ACDXPSettings.navToSyncedSitesMenu();
+
+			ACDXPSettings.assignSiteToExistingProperty(
+				assignedPropertyName = "Liferay DXP",
+				siteName = "Site Name");
+		}
+	}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:63]: hey @Tim-Cao , I just noticed in this test case you should go first to AC to create a new property and then return to DXP and proceed with the connection assigning the Site to that property.

Could you please double-check? 

Thanks!

Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:63]: You're right. I will update. Btw, I've close this PR. https://github.com/brianchandotcom/liferay-portal/pull/106531

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**